webbpsf/tests/test_opds.py

```diff
@@ -22,3 +29,49 @@ def test_enable_adjustable_ote():
 
     assert ote.rms() > rms*10, "Huge piston offset didn't make the WFE much worse"
 
+
+
+GLOBAL_FOCUS = [-0.018251043541410904e-08]
+GLOBAL_FOCUS_SCALED = [-0.018251043541410904e-08 * 0.5]
+COEFFS_A1 = np.array([-3.52633363e-09, -2.90050902e-09, 1.25432196e-09, -7.43319098e-12,
+                      -5.82462948e-11, -1.27115922e-10, -1.91541104e-12, 3.64760396e-11,
+                      4.97176630e-13])
+thermal_model_parameters = ([1 * u.day, 'SM', GLOBAL_FOCUS, None],
+                            [1 * u.day, 'SM', GLOBAL_FOCUS_SCALED, .5],
+                            [1440, 'SM', GLOBAL_FOCUS, None],
+                            [1 * u.day, 'A1', COEFFS_A1, None],
+                            [0.0 * u.day, 'SM', [0.0], None],
+                            [0.0 * u.day, 'A1', np.zeros(9), None],
+                            [1.0 * u.day, 'D1', [0.0], None])
```

Comment:
That is not a typo, that is checking to make sure that the code reacts properly when giving it a segment that doesn't exist (we don't want it to return an empty list - which is what it does if you ask it for a segment that doesn't exist - because that will mess things up).